resources/md/guestbook.md

```diff
@@ -496,7 +496,7 @@ We'll create a new controller that will be responsible for saving new messages i
           (empty? message)
           (assoc-in [:flash :errors :message] "message is required"))
         (do
-          (query-fn :save-message! {:name name :message message})
+          (query-fn :save-message! {:name name :message message :timestamp (java.util.Date.)})
```

Comment:
Without this, there was no way to set the timestamp through the form's POST operation (after making the previous change).